recipes/hamip/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "HaMiP" %}
+{% set version = "0.0.3.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 73fb10ce12e42e389f93a23424190e5eb4659c7e670bb6fd4c3b278c5002f366
+
+build:
+  number: 0
+  entry_points:
+    - HaMiP=HaMiP.HaMiP:main
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+
+requirements:
+  host:
+    - matplotlib-base
+    - pandas
+    - pip
+    - python >=3
+    - pyyaml
+  run:
+    - matplotlib-base
+    - pandas
+    - python >=3
+    - pyyaml
+    - bedtools
+    - ucsc-fetchchromsizes
+    - r-base =4
```

Comment:
if r-base is strictly needed in version 4 ... can we use >=4

Reply:
Hi @bgruening , yes, r-base dependency has been fixed using >=4. Please help review my PR again. Thank you very much.